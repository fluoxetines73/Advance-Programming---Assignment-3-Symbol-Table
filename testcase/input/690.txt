INSERT a string
BEGIN
END
ASSIGN a 'HelloWorld'
LOOKUP a
PRINT
RPRINT
LOOKUP a
BEGIN
ASSIGN a 'Test123'
INSERT c string
END