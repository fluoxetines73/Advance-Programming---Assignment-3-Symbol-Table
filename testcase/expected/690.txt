success
success
0
a//0
a//0
0
success
success